.taskcluster.yml

```diff
@@ -51,478 +51,474 @@ tasks:
         else: 'https://api.coverage.moz.tools'
       else: 'https://api.coverage.moz.tools'
 
-    taskboot_image: "mozilla/taskboot:0.2.1"
+    provisionerId:
+      $if: 'taskcluster_root_url == "https://firefox-ci-tc.services.mozilla.com"'
+      then:
+          $if: 'tasks_for == "github-push"'
+          then: 'code-analysis-3'
+          else: 'code-analysis-1'
+      else: proj-relman
+
+    workerType:
+      $if: 'taskcluster_root_url == "https://firefox-ci-tc.services.mozilla.com"'
+      then: linux
+      else: ci
+
+    taskboot_image: "mozilla/taskboot:0.2.2"
   in:
-    # TODO: move the bot docker build flow on Firefox CI Taskcluster
-    # when https://phabricator.services.mozilla.com/D63876 lands
-    $if: "taskcluster_root_url == 'https://firefox-ci-tc.services.mozilla.com'"
-    then: []
-    else:
-    - taskId: {$eval: as_slugid("check_lint")}
-      provisionerId: proj-relman
-      workerType: ci
-      created: {$fromNow: ''}
-      deadline: {$fromNow: '1 hour'}
-      payload:
-        maxRunTime: 3600
-        image: python:3.8
-        command:
-          - sh
-          - -lxce
-          - "git clone --quiet ${repository} /src && cd /src && git checkout ${head_rev} -b checks &&
-            cd /src/bot && pip install --disable-pip-version-check --no-cache-dir --quiet -r requirements-dev.txt &&
-            cd /src/backend && pip install --disable-pip-version-check --no-cache-dir --quiet -r requirements-dev.txt &&
-            cd /src && pre-commit run -a --show-diff-on-failure"
-      metadata:
-        name: "Code Coverage checks: linting"
-        description: Check code style with pre-commit hooks
-        owner: bastien@mozilla.com
-        source: https://github.com/mozilla/code-coverage
+    $flatten:
+      $match:
+        # Always run those tasks
+        "true":
+          - taskId: {$eval: as_slugid("check_lint")}
```

Comment:
Maybe we could run this task only on one of the two, and drop it from the dependencies of the other tasks

Reply:
It's cheap & convenient for now. Ideally everything would be on firefox-ci (like on code-review)